frontend/src/features/admin-form/responses/IndividualResponsePage/IndividualResponseNavbar.tsx

```diff
@@ -126,6 +130,12 @@ export const IndividualResponseNavbar = (): JSX.Element => {
 
   const isAdminPrintPdfEnabled = useFeatureIsOn(featureFlags.adminPrintPdf)
 
+  const printableResponseRef = useRef<HTMLDivElement>(null)
+  const reactToPrintFn = useReactToPrint({
+    contentRef: printableResponseRef,
```

Comment:
currently does not support bulk download as it relies on the [print window](https://github.com/MatthewHerbst/react-to-print?tab=readme-ov-file#can-react-to-print-be-used-to-download-a-pdf-without-using-the-print-preview-window). but this can be supported via introducing a dedicated pdf generator library eg, pdfjs / react-print renderer

Reply:
product has noted bulk download as not required for now